app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -7,7 +7,7 @@ def trademark(copy)
       case copy
       when "Forever"
         copy.upcase
-      when "Forever Historian", "Forever Valet", "Forever Live!", "Forever Print"
+      when "Forever Historian", "Forever Live!", "Forever Print"
```

Comment:
same as comment on colors, we want to remove the public representation but let's wait until the business makes a decision to pull the underlying support for it